src/config.rs

```diff
@@ -1105,7 +1110,9 @@ impl Config {
 
         // Fill any missing with defaults
         let config = builder.build();
-        validate_config(&config)?;
+        if !SKIP_CONFIG_VALIDATION.load(Ordering::SeqCst) {
```

Comment:
Using Relaxed ordering is more than enough for our use cases
```suggestion
        if !SKIP_CONFIG_VALIDATION.load(Ordering::Relaxed) {
```

Reply:
Ok done.